code/modules/projectiles/gun.dm

```diff
@@ -394,16 +394,16 @@
 		if (playemote)
 			if(reflex)
 				user.visible_message(
-					span("reflex_shoot","<b>\The [user] fires \the [src][pointblank ? " point blank at \the [target]":""] by reflex!</b>"),
-					span("reflex_shoot", "You fire \the [src] by reflex!"),
+					SPAN_DANGER("<b>\The [user] fires \the [src][pointblank ? " point blank at \the [target]" : ""] by reflex!</b>"),
+					SPAN_DANGER("You fire \the [src] by reflex!"),
 					"You hear a [fire_sound_text]!"
-					)
+				)
 			else
 				user.visible_message(
-					span("danger","\The [user] fires \the [src][pointblank ? " point blank at \the [target]":""]!"),
-					span("warning","You fire \the [src]!"),
+					SPAN_DANGER("\The [user] fires \the [src][pointblank ? " point blank at \the [target]" : ""]!"),
+					SPAN_DANGER("You fire \the [src]!"),
```

Comment:
shouldn't this show pointblank too

Reply:
dunno, which one's better?

Reply:
from a user feedback standpoint, it's probably a good idea to tell them if their damage multiplier ability worked, otherwise they might think it's busted